lib/utils.js

```diff
@@ -7,6 +7,12 @@ function isPlainObject(value) {
 }
 exports.isPlainObject = isPlainObject;
 
+// Support dynamic require within webpack bundle.
+if (typeof __non_webpack_require__ === 'undefined') {
+  var __non_webpack_require__ = require;
```

Comment:
Hmm, I think the presence of this `var` is going to ensure that `__non_webpack_require__ === 'undefined'` is always true. How about this:
```js
exports.require = typeof __non_webpack_require__ === 'undefined'
  ? require : __non_webpack_require__;
```